### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Screen readers] [MAS3.1.1][Lang attribute is not defined for the Script lab page]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#718]
**User Experience:**
User with vision disability  will face difficulty ,as the lang attribute is not defined and if the page isn’t in default language, then the screen reader might not announce the page text correctly.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0

**Attachments:**
![mas3 1 1 lang](https://user-images.githubusercontent.com/33054846/32720723-66a4fb08-c88a-11e7-9a5d-cf3532b8e926.png)

**Repro Steps:**
1. Open the URL: https://script-lab.azureedge.net/#/edit/excel
2. Verify the lang attribute for the page using F12.

**Current Behavior:**
Lang attribute is not defined.

**Expected Behavior:**
Lang attribute should be defined ,otherwise screen reader might not read the text correctly when the language change from the default.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={eb8515c6-18d7-4af0-b47e-c46958b1b50e}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=3.1.1
